roles/bibdata/tasks/main.yml

```diff
@@ -6,3 +6,19 @@
 
 - name: bibdata | include tasks to mount disks
   ansible.builtin.include_tasks: mounts.yml
+
+- name: bibdata | update rubygems
+  ansible.builtin.command: sudo gem update --system 3.5.7
```

Comment:
Does it still work if we use `become:` instead of sudo?

Reply:
I believe so! 

I also tried to make the "changed_when" more informative